{"name": "test_no_such_login", "status": "passed", "description": "Проверка невозможности залогиниться с несуществующим аккаунтом. Получаем ошибку 404", "steps": [{"name": "Логин курьера", "status": "passed", "parameters": [{"name": "payload", "value": "{'login': 'wxftohilcu', 'password': 'wcrcxgdunr', 'firstName': 'vngtpwvfat'}"}], "start": 1718006394968, "stop": 1718006395036}], "start": 1718006394968, "stop": 1718006395036, "uuid": "54e4e2c0-187e-44f4-822b-4c53c325613c", "historyId": "f8ba477e2ecf698f789d7f765c2b2fc5", "testCaseId": "f8ba477e2ecf698f789d7f765c2b2fc5", "fullName": "tests.test_login_courier.TestLoginCourier#test_no_such_login", "labels": [{"name": "feature", "value": "Ручка /api/v1/courier/login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "NB-MP25VCCB"}, {"name": "thread", "value": "18164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}